scripts/kind-e2e/e2e.sh

```diff
@@ -184,6 +184,8 @@ with_context cluster2 connectivity_tests
 
 # dataplane E2E need to be modified for globalnet
 if [[ $globalnet = false ]]; then
+    unset KUBECONFIG
```

Comment:
This is fixed in https://github.com/submariner-io/submariner-operator/pull/314.

Reply:
Awesome, can we get that merged then?